File: app-java/src/main/java/com/java/bilibili/widget/CardFrameLayout.java
package com.java.bilibili.widget;

import android.content.Context;
import android.support.v7.widget.CardView;
import android.util.AttributeSet;
public class CardFrameLayout extends CardView {
    public CardFrameLayout(Context context) {
        super(context);
        init();
    }

    private void init() {
        setUseCompatPadding(true);
    }

    public CardFrameLayout(Context context, AttributeSet attrs) {
        super(context, attrs);
        init();
    }

    public CardFrameLayout(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        init();
    }
}

File: app-java/src/main/java/com/java/bilibili/widget/CardFrameLayout.kt
package com.kotlin.bilibili.widget

import android.content.Context
import android.support.v7.widget.CardView
import android.util.AttributeSet

class CardFrameLayout : CardView {

    constructor(context: Context) : super(context) {
        init()
    }

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs) {
        init()
    }

    constructor(context: Context, attrs: AttributeSet, defStyleAttr: Int) : super(context, attrs, defStyleAttr) {
        init()
    }

    private fun init() {
        useCompatPadding = true
    }
}
